### PR TITLE
Interpret Colordrawable objects

### PR DESCRIPTION
**Pull request by joeatwork**
A special case to accomodate background colors.

### CODE REVIEW

File: src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java [hunk 1]
```diff
@@ -342,7 +343,13 @@ private Object convertArgument(Object jsonArgument, String type, List<String> as
             } else if ("float".equals(type) || "java.lang.Float".equals(type)) {
                 return ((Number) jsonArgument).floatValue();
             } else if ("android.graphics.drawable.Drawable".equals(type)) {
```
Comment (yinfeiru) [inline src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java]: can you do
`else if ("android.graphics.drawable.Drawable".equals(type) || "android.graphics.drawable.BitmapDrawable".equals(type)) {...}`

File: src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java [hunk 2]
```diff
@@ -351,11 +358,11 @@ private Object convertArgument(Object jsonArgument, String type, List<String> as
         }
     }
 
-    private Drawable readDrawable(JSONObject description, List<String> assetsLoaded)
```
Comment (yinfeiru) [inline src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java]: Why do you change readDrawable to readBitmapDrawable?

Comment (joeatwork) [inline src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java]: I think choosing to interpret Drawable as a particular subclass was a mistake I made in the earlier protocol- the UI is smart enough to pick a concrete class (like BitmapDrawable or ColorDrawable) and it means that if we product NinepatchDrawables or ScaleDrawables or whatever in the future, it'll be a bit less confusing.

File: src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java (hunk 1 above)
Comment (joeatwork) [inline src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java]: I wanted a good spot for the explanatory comment below- "This case is weird", as separate from the BitmapDrawable case.

Comment (yinfeiru) [inline src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java]: sounds good

**Comment by yinfeiru** [comment]
Looks good to me

**Pull request merged**